plugins/dell-k2/fu-dell-k2-ec-hid.h

```diff
@@ -13,6 +13,9 @@
 #define DELL_K2_EC_HID_CMD_FWUPDATE    0xAB
 #define DELL_K2_EC_HID_EXT_FWUPDATE    0x80
 #define DELL_K2_EC_HID_SUBCMD_FWUPDATE 0x00
+#define DELL_K2_EC_DEV_EC_CHUNK_SZ     160000
+#define DELL_K2_EC_DEV_ANY_CHUNK_SZ    180000
+#define DELL_K2_EC_DEV_NO_CHUNK_SZ     9999999
```

Comment:
not sure about this; don't you want `G_MAXSIZE`?

Reply:
Oh I should use it instead the ugly num, will fix.